tools/gyp/buildbot/buildbot_run.py

```diff
@@ -30,7 +21,8 @@
 
 def CallSubProcess(*args, **kwargs):
   """Wrapper around subprocess.call which treats errors as build exceptions."""
-  retcode = subprocess.call(*args, **kwargs)
+  with open(os.devnull) as devnull_fd:
+    retcode = subprocess.call(stdin=devnull_fd, *args, **kwargs)
```

Comment:
Assign a default value to `retcode` before this block. If the open fails or `subprocess.call` fails, the next `if` condition will raise a `NameError`.